backend/trojan_dining/models.py

```diff
@@ -42,6 +42,7 @@ class Meta:
 class Menu(models.Model):
         # for queries
         created_at = models.DateField(auto_now_add=True)
+        menu_date = models.DateField(default = None)
```

Comment:
this is already in the Menu model, we can just call it `date`

Reply:
changed it to date